src/containers/Ratings/Ratings.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import { Mutation } from 'react-apollo';
+import gql from "graphql-tag";
+import { connect } from 'react-redux';
+import { handleData } from '../../thunks/handleData'
+import { postShotData } from '../../queries/queries'
+import { setLastShot } from '../../actions'
+
+class Ratings extends Component {
+  state = {
+    loading: true,
+    result: 3,
+    currClub: 0,
+  }
+
+  componentDidMount() {
+    const { club } = this.props
+    let currClub = 0
+    switch(club) {
+      case 'wedges':
+        currClub = 1
+        break
+      case 'irons':
+        currClub = 2
+        break
+      case 'woods':
+        currClub = 3
+        break
+    }
+    this.setState({currClub})
+  }
+
+  
+  updateResult = (e) => {
+    const { value } = e.target
+    this.setState({ result: parseInt(value) })
+  }
+
+  handleClick = () => {
+    const { currClub, result} = this.state
+    this.props.handleData(setLastShot, postShotData(currClub, result))
+    this.props.changeFilter('home')
+  }
+
+
+  // getQuery = () => {
```

Comment:
We talked about this and I agree that we should get our queries working minus apollo for the time being.  Let's get mvp up and running and we can come back and refactor